galleries/users_explain/axes/legend_guide.py

```diff
@@ -68,6 +71,18 @@
     ax.legend([line_up, line_down], ['Line Up', 'Line Down'])
 
 
+If the handles are not directly accessible, for example when using some
+`Third-party packages <https://matplotlib.org/mpl-third-party/>`_, they can be accessed
+as follows::
+
+    my_map = {'Line Up':'Up', 'Line Down':'Down'}
+
+    handles, labels = ax.get_legend_handles_labels()
+    ax.legend(handles, [my_map[l] for l in labels])
+
+Here we use a dictionary to ensure that we rename the correct entry.
```

Comment:
IMHO it's helpful to explicitly spell out the procedure so that users don't have to infer that from the example.

```suggestion
via `.Axes.get_legend_handles_and_labels` and fed into `.Axes.legend` explicitly.
Here we use a dictionary to rename existing labels::

    my_map = {'Line Up':'Up', 'Line Down':'Down'}

    handles, labels = ax.get_legend_handles_labels()
    ax.legend(handles, [my_map[l] for l in labels])
```